src/role_scout.js

```diff
@@ -112,7 +112,12 @@ roles.scout.execute = function(creep) {
       // }
       return false;
     }
-
+    if (creep.isStuck()) {
+      creep.moveRandom();
+      creep.say('ImStuck', true);
+      creep.log(JSON.stringify(creep.memory.last) + ' ' + JSON.stringify(creep.isStuck()));
+      return false;
```

Comment:
`return true`
This method handles the creep so `true`. Otherwise other methods would be called and interfere with your logic.